A.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```

Comment:
[Optional]

- [ ] It's not required to focus on the content of the HTML and CSS files right now however if you want the CSS to work on your HTML elements, kindly add a link to your CSS stylesheet in the HTML `head` section 💯